src/types.ts

```diff
@@ -1,6 +1,21 @@
 import Augur from "augur.js";
 import * as Knex from "knex";
 
+export enum REPORTING_STATE {
```

Comment:
Looks like the TS convention for Enums is camel case (because they're a user-defined type, like a class), so update to `ReportingState` and then use that type where you would have used `string`